components/sup/src/manager/census.rs

```diff
@@ -14,37 +14,37 @@
 
 use std::ops::{Deref, DerefMut};
 use std::collections::HashMap;
-use std::str::FromStr;
+use std::str::{self, FromStr};
 
 use hcore::package::PackageIdent;
 use hcore::service::ServiceGroup;
-use butterfly::rumor::service::Service as ServiceRumor;
-use butterfly::rumor::election::{Election as ElectionRumor, Election_Status};
+use butterfly;
+use butterfly::rumor::{Service as ServiceRumor, Election as ElectionRumor};
```

Comment:
If we were (and we should soon!) generating docs for Butterfly the value of this might be more apparent. The generated docs would show the same content regardless of the module path that you searched for the rumor struct on.

Here's a great example in serde. When reading the docs you can check out the [re-exports at the top of the crate](https://docs.serde.rs/serde/#reexports). It makes the API surface clear to the reader without asking them to read each submodule. Your docs are also easier to write because you don't need to write public documentation for each submodule, instead you can write it at the container module level and explain to people how to use your API